app/views/components/inputs/input_accessible_autocomplete.scala.html

```diff
@@ -40,14 +45,22 @@
   else if (maxLength >= LowerWidthBand && maxLength < UpperWidthBand) "govuk-input--width-30"
   else "govuk-input--width-20"
 }
-
-<div id="@(fieldNameWithoutBrackets)-container" class="autocomplete__wrapper @widthClass">
-    <select class="govuk-form-group" id="@(fieldNameWithoutBrackets)" name="@field.name" value="@field.value" data-gov-select-autocomplete="true">
-        <option value="">@emptySelectValue</option>
-        @items.map { i =>
-            <option value="@i.value" @if(selectedValue == i.value) {selected="selected"}>@i.label</option>
-        }
-    </select>
+<div class="govuk-grid-row">
+    <div class="@rightElement.map(_ => "govuk-grid-column-two-thirds").getOrElse("govuk-grid-column-full")">
+        <div id="@(fieldNameWithoutBrackets)-container" class="autocomplete__wrapper @widthClass">
+            <select class="govuk-form-group" id="@(fieldNameWithoutBrackets)" name="@field.name" value="@field.value" data-gov-select-autocomplete="true">
```

Comment:
Not sure about having the same `id` for two different elements, even if they are parent-child. But maybe it does not harm.

Reply:
They are not the same, one is "@(fieldNameWithoutBrackets)-container" and the inner element is just "@(fieldNameWithoutBrackets)"

Reply:
You are right. I missed `-container` :+1: